.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: RPC Tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+#     strategy:
+#       matrix:
+#         python-version: [3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: python3 -m pip install flake8
+      - run: cp client/config/config.example.js client/config/config.js
+      - run: npm install
+      - run: npm run lint
```

Comment:
Re-tested with proper failure code, and it works. Love the error log much better than travis, looks more cleaner! 
![image](https://user-images.githubusercontent.com/29296452/110422957-17837a00-8055-11eb-8b86-378d641658c7.png)